### PR TITLE
Rating n snackbar

### PR DESCRIPTION
**Pull request by ivinayakg**

### CODE REVIEW

File: COMPONENTS/rating/index.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+@import url(/CSS/global.css);
+
+.rating_comp {
```
Comment (irajbhinde) [inline COMPONENTS/rating/index.css:3]: can remove the unused css here

File: COMPONENTS/rating/index.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  /* margin: 4px; */
+  display: flex;
+  width: max-content;
+  height: max-content;
+}
+
+.rating_star {
+  width: max-content;
+  height: max-content;
+  padding: 0px;
```
Comment (irajbhinde) [inline COMPONENTS/rating/index.css:17]: margin & padding 0 can be applied to body instead of assigning to class

**Review by irajbhinde — COMMENTED** [review]
Looks good!

File: COMPONENTS/snackbar/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  });
+
+  target.addEventListener("click", (e) => {
+    console.log(e.target);
+    if (snackbar.className.includes("snackbar--active")) {
+      return;
+    } else {
+      snackbar.classList.add("snackbar--active");
+    }
+  });
```
Comment (darshandamre) [inline COMPONENTS/snackbar/index.js:20]: you can remove this `console.log`
and the if...else can be written as
```js
if (snackbar.className.includes("snackbar--active")) return;

snackbar.classList.add("snackbar--active");
```

Comment (ivinayakg) [reply COMPONENTS/snackbar/index.js:20]: thank you i will surely do that

**Comment by ivinayakg** [comment]
> Looks good!

thank you

**Pull request merged**